<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Thong_Tin_Thi</class>
 <widget class="QDialog" name="Thong_Tin_Thi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="ThongTin">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>421</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:20pt; font-weight:700; color:#ff0000;&quot;&gt;Thông Tin Thi Trắc Nghiệm&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="DSachMonHoc">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>391</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
	border: 2px solid gray;
	border-radius: 20px;
	background-color: #F0F0F0;
	color: black;
	font-size: 16px;
text-align: center;
}

QComboBox::drop-down {
width: 30px;
subcontrol-origin: padding;
subcontrol-position: top right;
border: 2px solid gray;
border-radius: 10px;
background-color: #F0F0F0;
}

QComboBox::down-arrow {
    image: url(:/logo/down-arrow (1).png); /* Đường dẫn hình ảnh mũi tên */
    width: 20px;
    height: 20px;
}

QComboBox QAbstractItemView {
    border: 2px solid gray;
    border-radius: 10px;
    background-color: white;
    selection-background-color: lightblue;  /* Màu nền của mục được chọn */
    selection-color: black;  /* Màu chữ của mục được chọn */
    padding: 5px;
	font-size: 12px;
}

</string>
   </property>
   <property name="placeholderText">
    <string>                           Danh Sách Môn Học</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>230</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
    border: 2px solid gray;
    border-radius: 20px;
    background-color: #F0F0F0;
    color: black;
    font-size: 16px;
    padding: 2px;  /* Thêm khoảng cách giữa viền và nội dung */
    qproperty-alignment: 'AlignCenter';  /* Căn giữa văn bản */
}

QSpinBox::up-button, QSpinBox::down-button {
    width: 0px;
    height: 0px;
    border: none;
}


</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid gray;
	border-radius: 20px;
	background-color: #F0F0F0;
	color: black;
	font-size: 14px;
text-align: center;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#000000;&quot;&gt;Số Câu Hỏi Thi:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>300</y>
     <width>261</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid gray;
	border-radius: 20px;
	background-color: #F0F0F0;
	color: black;
	font-size: 16px;
text-align: center;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt; color:#000000;&quot;&gt;Số Phút Thi:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>300</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QSpinBox {
    border: 2px solid gray;
    border-radius: 20px;
    background-color: #F0F0F0;
    color: black;
    font-size: 16px;
    padding: 2px;  /* Thêm khoảng cách giữa viền và nội dung */
    qproperty-alignment: 'AlignCenter';  /* Căn giữa văn bản */
}

QSpinBox::up-button, QSpinBox::down-button {
    width: 0px;
    height: 0px;
    border: none;
}


</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DangNhapButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>370</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border: 2px solid gray;
border-radius: 20px;
background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);
font-size: 18px;
}

QPushButton:hover {
    background-color: rgb(200, 0, 0);  /* Màu nền khi hover (đậm hơn) */
    cursor: pointer;  /* Con trỏ chuột sẽ chuyển thành hình bàn tay khi hover */
}

</string>
   </property>
   <property name="text">
    <string>Hoàn Thành</string>
   </property>
  </widget>
  <widget class="QLabel" name="MSSV">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>90</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid gray;
	border-radius: 10px;
	background-color: #F0F0F0;
	color: black;
text-align: center;
font-size: 12pt;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;MSSV: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="TEN">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>90</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid gray;
	border-radius: 10px;
	background-color: #F0F0F0;
	color: black;
text-align: center;
font-size: 12pt;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Tên: &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="LoiDSach">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>200</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LoiCauHoi">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="LoiThoiGian">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>340</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
